articles/connections/passwordless/sms-gateway.md

```diff
@@ -77,14 +78,43 @@ You can then send the updated configuration to the Management API using the [PAT
 
 After updating the connection for any user that signs up or authenticates using the Passwordless SMS connection the following payload will be sent to your SMS gateway:
 
+```
+{
+  "recipient": "+1 399 999",
+  "body": "Your verification code is: 12345",
+  "sender": "+1 234 567"
+}
+```
+
+If you set the `forward_req_info` property to **true**, the gateway will also receve information from the HTTP request that initiated the Passwordless process.
+
 ```
 {
   "recipient": "+1 399 999",
   "body": "Your verification code is: 12345",
   "sender": "+1 234 567",
+  "req" : { 
+      "ip" : "167.56.227.117",
```

Comment:
How will this value be calculated in Auth0 tenant?

Reply:
How does the tenant determine what the client IP is? Does it use x-forwarded-for when present, and if so does that mean any client can override client IP or change from call to call?